meta-resin-common/recipes-support/resin-mounts/resin-mounts.bb

```diff
@@ -32,6 +32,7 @@ do_install () {
     install -d ${D}/mnt/conf
     install -d ${D}/mnt/data
     install -d ${D}/mnt/boot
+    install -d ${D}/etc/docker
```

Comment:
Please move this to this commit instead: https://github.com/resin-os/meta-resin/pull/293/commits/011b27a1bc2925e1f9824b07c2a1d0294b0f4a6e